roles/srv-announcer/defaults/main.yml

```diff
@@ -0,0 +1,51 @@
+# NOTE: variables set here can - and potentially should - be overridden when invoking the role
```

Comment:
This comment is not news and can be removed. Anyone reading the ansible documentation knows that variables under defaults/main can be overriden.

Reply:
In my opinion it's valuable to put a fact from the docs into the code that actually explains why this file exists and how its content should be understood. If you think, for example, of on-prem or just people using what we provide w/o specific domain knowledge.

Also, I'm a bit confused, because you signed of [#1](https://github.com/wireapp/ansible-sft/pull/1) which contained a [similar comment](https://github.com/wireapp/ansible-sft/pull/1/files#diff-ee5d19d9692ad3b7ab8374167a65ca48R1) w/o highlighting this.

Reply:
If you think documentation for end users is needed, I think a good place would be on docs.wire.com, and/or inside the README. If you know enough ansible to know that defaults/main.yml contain overridable variables, and then open that file, you don't need to be told this anymore. So I feel this comment isn't very useful.

> Also, I'm a bit confused, because you signed of #1 which contained a similar comment w/o highlighting this.

I generally only comment on the most important issues on a given PR and let less important things pass. Also, I don't necessarily review every line of code or comment in detail. I noticed it here, so I commented on it. Having received an approval for 659 lines of code doesn't mean I fully agree 100% with each of those lines of code.

But it's a comment, so I won't continue a discussion here anymore, if you feel strongly to keep it, keep it.